approval_polls/templates/approval_polls/my_polls.html

```diff
@@ -0,0 +1,52 @@
+{% extends 'base.html' %}
+
+{% block content %}
+  <div  class='row-fluid top-buffer'>
+      <div class='col-md-12'>
+          <div class='well well-sm'>
+            Polls created by {{ user.username }}:
+        </div>
+    </div>
+  </div>
+  {% if latest_poll_list %}
+    <div class='row-fluid top-buffer'>
+      <div class='col-md-12'>
+        {% for poll in latest_poll_list %}
+          <div class='well well-sm'>
+            <a href="{% url 'approval_polls:detail' poll.id %}">{{ poll.question }}</a>
+            &nbsp;&nbsp;&nbsp;<a> remove </a> 
```

Comment:
As the buttons tend to stick together when pulled right, add a css class in style.css called `right-buffer` which sets the `margin-right` to 5px. Use this for the last two links.